contracts/mocks/TimeFrameMock.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.5.0;
+
+import "../lifecycle/TimeFrame.sol";
+
+
+contract TimeFrameMock {
+    using TimeFrame for TimeFrame.Epoch;
+
+    TimeFrame.Epoch private _epoch;
+
+    constructor(uint256 start, uint256 end) public {
+        _epoch = TimeFrame.Epoch(start, end);
```

Comment:
Nice, I didn't know about this syntax. Can we add custom code to the constructor?